Add SharpVk

FacticiusVir opened the pull request.

daveaglick commented:
Thanks! Would it be okay if we used the tag "Graphics" instead of the more specific "Vulkan" and "SPIR-V"? I'm trying to limit tag explosion and stick to existing more general ones where possible (though "3D" seems like a good one to add). There's a fine line - too many specific tags for every technology and the tags themselves become useless from overload.

daveaglick commented:
Perfect, thanks! The site is updated every 2 hours, so this change will appear on the sometime in that window.

FacticiusVir commented:
Sure thing, I've removed the technology-specific tags and added the relevant tags from your project search page - and also corrected the missing .yml extension :-D
